egs/wsj/s5/utils/data/get_utt2dur.sh

```diff
@@ -68,7 +70,10 @@ else
       echo  "$0: wav-to-duration is not on your path"
       exit 1;
     fi
```

Comment:
Add code to check if there are any sox command with keyword "speed"; if so, set read_entire_file=true.